lib/WP_Auth0_Import_Settings.php

```diff
@@ -165,7 +165,7 @@ public function export_settings() {
 		header( "Content-Disposition: attachment; filename=auth0_for_wordpress_settings-$name.json" );
 		header( 'Pragma: no-cache' );
 
-		$settings = $this->a0_options->get_options();
+		$settings = get_option( $this->a0_options->get_options_name() );
```

Comment:
Get settings from the DB instead of in-memory array to skip exporting constant-based settings.